html2text/__init__.py

```diff
@@ -181,11 +184,7 @@ def handle_charref(self, c):
         self.handle_data(charref, True)
 
     def handle_entityref(self, c):
```

Comment:
@lukasz-madon makes a good point: I could pass `&lt;script&gt;alert('hacking you');&lt;/script&gt;` and that'll be converted nicely into `<script>alert('hacking you')</script>`.  Probably not what we want.